src/i18n/en-events.ts

```diff
@@ -13,6 +13,7 @@ const eventsMessageDescriptors = {
       ${ColonyAndExtensionsEvents.RecoveryStorageSlotSet} {{initiator} submitted new value for slot {storageSlot}}
       ${ColonyAndExtensionsEvents.RecoveryModeExitApproved} {{initiator} approved exiting}
       ${ColonyAndExtensionsEvents.RecoveryModeExited} {{initiator} exited Recovery Mode}
+      ${ColonyAndExtensionsEvents.MotionCreated} {{initiator} {reputation} created a {motionTag}}
```

Comment:
So there's two things here:
1. I think we need to add reputation to all items in the feed that have an initiator, including the normal actions
2. I see the reputation being more integrated with the initiator, rather than being a separate entity

So in my head, you just have the initiator as a motion descriptor value:
```js
      ${ColonyAndExtensionsEvents.MotionCreated} {{initiator} created a {motionTag}}
```

Then when passing along the values, you just do something like this:
```js
    initiator: (
      <>
        <span className={styles.titleDecoration}>
          <FriendlyName user={initiator} autoShrinkAddress />
        </span>
        <div className={motionSpecificStyles.reputation}>
          <MemberReputation
            walletAddress={actionInitiator}
            colonyAddress={colony.colonyAddress}
          />
        </div>
      </>
    ),
```

Meaning you pass both the initiator and reputation in the same component